Dockerfile.builder

```diff
@@ -39,7 +39,7 @@ RUN apt-get update && \
 # Install Bazelisk
 #
 # renovate: datasource=github-releases depName=bazelbuild/bazelisk
-ENV BAZELISK_VERSION=v1.20.0
```

Comment:
looks like we still miss the updates in `Dockerfile.builder.tests` & `install_bazelisk.sh`?

Reply:
Yeah, checking now 🤔

Reply:
and the other missing piece is that the custommanagers also doesn't include all files and the regex doesn't match all cases.

Reply:
Thanks a lot for your help :+1: 

https://github.com/cilium/proxy/pull/1310

Reply:
Thanks! I assume you open a separate PR that handles `install_bazelisk.sh` (where the regex needs to be adjusted too)

Reply:
yup :+1: